lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -1437,11 +1437,18 @@ defmodule Kernel.SpecialForms do
       4
 
   In other words, `&(&1 * 2)` is equivalent to `fn x -> x * 2 end`.
-  Another example using a local function:
 
-      iex> fun = &is_atom(&1)
-      iex> fun.(:atom)
-      true
+  We can partially apply a remote function with placeholder:
+
+      iex> take_five = &Enum.take(&1, 5)
+      iex> take_five.(1..10)
+      [1, 2, 3, 4, 5]
+
+  Another example while using an imported or local function:
+
+      iex> first_elem = &elem(&1, 1)
```

Comment:
This doesn't work like in the example: `elem/2` in Elixir is 0-indexed so `first_elem` should be `&elem(&1, 0)`.